gstools/variogram/variogram.py

```diff
@@ -75,6 +77,17 @@ def vario_estimate_unstructured(
         the spatially distributed data
     bin_edges : :class:`numpy.ndarray`
         the bins on which the variogram will be calculated
+    angles : :class:`numpy.ndarray`
+        the angles of the main axis to calculate the variogram for in radians
+        angle definitions from ISO standard 80000-2:2009 
```

Comment:
Wow, I guess you're an engineer?! :-)

Reply:
Yep I am :-D

Reply:
Could you stick to the angles definition/explanation of the CovModel?
https://github.com/GeoStat-Framework/GSTools/blob/2c26d749ecd8554f10579bef4ac2d1f70266b5a9/gstools/covmodel/base.py#L73

Reply:
OK, yea that would keep things consistent. Sorry, didn't saw that before.

Reply:
Please add that these arguments are optional and that the angles can also be float